raiden/utils/typing.py

```diff
@@ -42,8 +42,7 @@
 T_ChannelState = int
 ChannelState = NewType('ChannelState', T_ChannelState)
 
-T_InitiatorAddress = bytes
```

Comment:
Fixed.

I didn't like the look of
```
T_InitiatorAddress = bytes
InitiatorAddress = NewType('InitiatorAddress', Address)
```
because why `bytes` on the first line and `Address` on the second line?

But now
```
T_InitiatorAddress = T_Address
InitiatorAddress = NewType('InitiatorAddress', Address)
```
works for me.